frontend/src/types/reducer/metrics.ts

```diff
@@ -12,11 +9,15 @@ interface MetricReducer {
 	metricsApplicationLoading: boolean;
 	error: boolean;
 	errorMessage: string;
-	dbOverView: DBOverView[];
-	externalService: ExternalService[];
 	topEndPoints: TopEndPoints[];
-	externalAverageDuration: ExternalAverageDuration[];
-	externalError: ExternalError[];
+	applicationRpsEndpoints: queryEndpointData[];
+	applicationErrorEndpoints: queryEndpointData[];
+	dbRpsEndpoints: queryEndpointData[];
+	dbAvgDurationEndpoints: queryEndpointData[];
```

Comment:
can we add 3 tab level loading for each tab

Reply:
Will address this issue in the new pr. Looking into it 👍